pkgs/servers/geospatial/t-rex/default.nix

```diff
@@ -22,7 +22,7 @@ rustPlatform.buildRustPackage rec {
     homepage = "https://t-rex.tileserver.ch/";
     changelog = "https://github.com/t-rex-tileserver/t-rex/blob/v${version}/CHANGELOG.md";
     license = licenses.mit;
-    maintainers = with maintainers; [ sikmir ];
+    maintainers = teams.geospatial.members;
     mainProgram = "t_rex";
     platforms = platforms.unix;
     broken = true;  # https://github.com/t-rex-tileserver/t-rex/issues/302
```

Comment:
https://github.com/t-rex-tileserver/t-rex/issues/302 fixed in upstream, perhaps we can apply the patch and unbreak.

Reply:
Yes, I agree. Do you want to do it in this PR or in a new one ?

Reply:
This works for me:
```patch
diff --git i/pkgs/servers/geospatial/t-rex/default.nix w/pkgs/servers/geospatial/t-rex/default.nix
index c96d8a8247f5..b06d4061ffd0 100644
--- i/pkgs/servers/geospatial/t-rex/default.nix
+++ w/pkgs/servers/geospatial/t-rex/default.nix
@@ -2,16 +2,16 @@

 rustPlatform.buildRustPackage rec {
   pname = "t-rex";
-  version = "0.14.3";
+  version = "0.15.0-alpha3";

   src = fetchFromGitHub {
     owner = "t-rex-tileserver";
     repo = pname;
     rev = "v${version}";
-    hash = "sha256-LUVk5li2cl/LKbhKOh6Bbwav0GEuI/vUbDPLn7NSRIs=";
+    hash = "sha256-oZZrR86/acoyMX3vC1JGrpc8G+DEuplqfEAnaP+TBGU=";
   };

-  cargoHash = "sha256-I4QmjTTKUp9iugEwzM0xCcNLvF5ozeBdYmbi8sytY88=";
+  cargoHash = "sha256-nxq4mX2Sy6Hyi8tA2CQsQwISB/kau4DEkAgIm4SvGns=";

   nativeBuildInputs = [ pkg-config ];

@@ -25,6 +25,5 @@ rustPlatform.buildRustPackage rec {
     maintainers = teams.geospatial.members;
     mainProgram = "t_rex";
     platforms = platforms.unix;
-    broken = true;  # https://github.com/t-rex-tileserver/t-rex/issues/302
   };
 }
```

Reply:
https://github.com/NixOS/nixpkgs/pull/295097